scripts/servers_test.py

```diff
@@ -1035,3 +1035,24 @@ def test_managed_webdriverio_with_explicit_args(self) -> None:
         self.assertIn('--params.devMode=False', program_args)
         # From suite='full'.
         self.assertIn('--suite abc', program_args)
+
+    def test_managed_acceptance_test_server_with_explicit_args(self) -> None:
+        popen_calls = self.exit_stack.enter_context(self.swap_popen())
+
+        puppeteer_acceptance_tests_dir_path = os.path.join(
+            common.CURR_DIR, 'core', 'tests', 'puppeteer-acceptance-tests')
+        spec_dir_path = os.path.join(
+            puppeteer_acceptance_tests_dir_path, 'spec')
+        test_file_path = os.path.join(spec_dir_path, 'test')
+
+        self.exit_stack.enter_context(servers.managed_acceptance_tests_server(
+            suite_name='test',  # pylint: disable=line-too-long
+            stdout=subprocess.PIPE))
+        self.exit_stack.close()
+
+        self.assertEqual(len(popen_calls), 1)
+        self.assertEqual(
+            popen_calls[0].kwargs, {'shell': True, 'stdout': subprocess.PIPE})
+        program_args = popen_calls[0].program_args
+        # From suite.
```

Comment:
What does this mean?

Reply:
No, that has no particular meaning. It's just that we are asserting the suite params to be equal. We could also have added assertion for the stdout param but since it remains unchanged i didn't add it. So i might as well remove that comment, it doesn't serve any purpose there